tests/js/spec/views/dashboardsV2/gridLayout/detail.spec.jsx

```diff
@@ -467,59 +337,6 @@ describe('Dashboards > Detail', function () {
       expect(wrapper.find('AddWidget').exists()).toBe(true);
     });
 
-    it('opens custom modal when add widget option is clicked', async function () {
```

Comment:
Maybe I should keep this and the next test in 🤔  Technically tests that I've put the right handlers on the add button

Reply:
yeah, I guess we can leave them in